rollup.config.js

```diff
@@ -0,0 +1,21 @@
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+// import { terser } from 'rollup-plugin-terser';
+
+// `npm run build` -> `production` is true
+// `npm run dev` -> `production` is false
+const production = !process.env.ROLLUP_WATCH;
+
+export default {
+  input: 'static/js/component-bundle-entry.js',
+  output: {
+    file: 'static/js/component-bundle.js',
+    format: 'iife', // immediately-invoked function expression — suitable for <script> tags
+    sourcemap: true
+  },
+  plugins: [
+    resolve(), // tells Rollup how to find date-fns in node_modules
```

Comment:
Is date-fns really used? It's not listed in package.json

Reply:
date-fns 2.0 publishes modules to npm. How do you feel about pre-releases?

Reply:
My bad, not intended to be here.